src/WebUI/Views/Shared/_Layout.cshtml

```diff
@@ -30,17 +30,17 @@
         </div>
     </header>
     <aside class="header-slide" id="js-hms">
-        <a class="header-slide--item">Placeholder 1</a>
-        <a class="header-slide--item">Placeholder 2</a>
-        <a class="header-slide--item">Placeholder 3</a>
+        <a class="header-slide--item" href="#">Placeholder 1</a>
+        <a class="header-slide--item" href="#">Placeholder 2</a>
+        <a class="header-slide--item" href="#">Placeholder 3</a>
         <div class="header-slide--separator"></div>
-        <a class="header-slide--item" href="https://github.com/codidact/co-design">on GitHub</a>
+        <a class="header-slide--item" href="https://github.com/codidact/core">on GitHub</a>
     </aside>
     <main class="container has-padding-4">
         @RenderBody()
     </main>
 
-    <footer class="border-top footer text-muted">
+    <footer class="has-padding-4 has-border-top-style-solid has-border-top-width-1 has-border-color-tertiary-050">
```

Comment:
Not sure I like this. Can we take the effect of all these classes on `footer` and write them into a default style for `footer`?

Reply:
That's definitely possible and probably a good idea. However we'd need to review, how our footer should look like (minimal like here, large as on SE...) before doing so.